packages/api/src/hooks/useCreateOtherCFDAccount.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+import useRequest from '../useRequest';
+import useInvalidateQuery from '../useInvalidateQuery';
+
+/** A custom hook that creates the Other CFD account. */
+const useCreateOtherCFDAccount = () => {
```

Comment:
I think we can combine this hook with `useTradingPlatformAccounts` just like `useSettings`. So , we can fetch trading account list and create new account with the same hook. cc: @farzin-deriv

Reply:
Is it ok to have an api with post and get request as we are already using invalidated to get the refreshed list after creation and moreover if we use the `post` and `get` in same api so we need to pass params like platform to create the correct account which is just like creating a virtual backend layer. Correct me if I am wrong.
Thanks

Reply:
I think its better to leave this independent as we have one hook each for `fetch` and `request`. If in the future, we add more stuff to `useTradingPlatformAccounts` or more platforms come in, its gonna be doing a lot of stuff which will lead us to split it then. 👀

Reply:
Exactly that's my concern here

Reply:
Alright 👀 . So, if we are not gonna combine it, I have another concern. So, based on the request type, this endpoint should be able to accepts some payloads and will be used for mutation right. But, why we are fetching data and not making any `post` request / mutation here? 😬

Reply:
Same as `useCreateMT5Account`. We are not doing any mutation 👀

Reply:
Should the mutation be done from the component side like the payload send from the component or should we add the useAuthorize and get all the details here? Payload will contains the platform name incase of other CFD account as platform will be send on the basis of which account needs to be created. (DerivEz, Deriv X, cTrader). It can be done for MT5 but still the password is coming from the frontend, so useAuthorize can be used there(component) too? 👍

Reply:
Yes @hamza-deriv, the payload should be in the component. Since this api needs many parameters, we can just pass it from the component instead. 
Something like this 
`const { mutate: createWallet } = useCreateWallet();`
`createWallet({
payload: { currency, account_type: currency_config?.is_crypto ? 'crypto' : 'doughflow' },
 })`

Reply:
@farhan-nurzi-deriv The `useSettings` case is a bit different, `set_settings` and `get_settings` both are the same endpoint but for some reason, BE is messed up so we hide it in the abstraction, But here in this case we are talking about 2 different endpoint, Better to keep them separate.

--- 

Also, we are in `web-socket`, There is no `post` or `get` or even `fetch` or `request` 😅 It's all there because our web socket API is not as it should be 👀